test/e2e/history.vim

```diff
@@ -24,3 +24,20 @@ function! s:suite.test_keepalt() abort
         \ 'foo.txt',
         \ 'Vaffle navigation should keep alternate file')
 endfunction
+
+function! s:suite.test_keepalt() abort
+  cd test/e2e/files
+  " Open file
+  e history/alt.txt
+
+  Vaffle
+
+  " edit another file via vaffle; alt.txt should remain the alternate
+  call search('foo.txt')
```

Comment:
It seems that `foo.txt` is not found. Should we set the current working directory to `test/e2e/files/history` instead of `test/e2e/files`, or replace `Vaffle` with `Vaffle history`?

Reply:
It does seem foo.txt was not being found.  How did you determine this? I'm new to themis and just assumed it would error out. Although I see now that was incorrect.

Reply:
I've pushed your suggested change.

Reply:
Thank you!

> How did you determine this?

I have tested it manually... open Vim, and then typed each command step by step. Maybe if we'd like to test it more strictly, we should also test an opened file with `bufname`, but it's enough for now.